ambry-store/src/main/java/com.github.ambry.store/StoreFindToken.java

```diff
@@ -78,13 +101,18 @@ private StoreFindToken(Type type, Offset offset, StoreKey key, UUID sessionId) {
     this.offset = offset;
     this.storeKey = key;
     this.sessionId = sessionId;
+    this.incarnationId = incarnationId;
     this.bytesRead = -1;
   }
 
   void setBytesRead(long bytesRead) {
     this.bytesRead = bytesRead;
   }
 
+  void setInclusive() {
```

Comment:
I am adding one more constructor for the journal. If you feel we can remove the default one for the journal, let me know.